Taskfile.yml

```diff
@@ -126,11 +126,10 @@ tasks:
   docker:build:
     desc: build docker image
     cmds:
-      - docker build -t $DOCKER_REGISTRY/$IMAGE_NAME:latest .
+      - docker build -t $DOCKER_REGISTRY/$IMAGE_NAME:$IMAGE_VERSION .
```

Comment:
Kubernetes always fetches `latest` so this tag is not ok for local development and testing at all.

Reply:
agreed!